# Object ID to grab; NOTE: Leave this blank to automatically detect the object that exists in the EEF location and grasp it
string obj_id
---
# Properties about the physical movement of the robot
taskit/MovementProperties mv_props
